bedrock/careers/templates/careers/listings.html

```diff
@@ -12,40 +12,33 @@
 {% block body_class %}listings-page{% endblock %}
 
 {% block careers_content %}
-<section class="mzp-c-hero mzp-t-dark page-hero">
-  <div class="mzp-l-content">
-    <div class="mzp-c-hero-body">
-      <h1 class="mzp-c-hero-title">Feel good about your work again</h1>
-
-      <div class="mzp-c-hero-desc">
-        <p>
-          Join us.
-        </p>
-      </div>
+<header class="c-careers-text-hero">
+  <div class="mzp-l-content c-text-hero-wrapper mzp-t-content-md">
+    <h1 class="c-text-hero-title">Feel good about your work again</h1>
+    <div class="c-text-hero-body">
+      <p>Bring your drive, your creativity, your big ideas and your new perspectives to make the difference we’re aiming for.</p>
     </div>
   </div>
-</section>
+</header>
 
-<section class="mzp-l-content">
-  <h2>Listings</h2>
+<section class="mzp-l-content mzp-t-content-xl">
+  <h2 class="c-careers-section-title">Listings</h2>
```

Comment:
**Discussion:** We mentioned that we should add a `margin-bottom` or a spacing to this title section (https://github.com/reemhamz/bedrock/pull/6#discussion_r1025746918), I think we should still do that now that you've fixed up the spacing in this PR. What do you think?